src/models-spec.js

```diff
@@ -255,7 +254,28 @@ describe('models', () => {
                 expect(jsonData.fields.type.requiredContent).toMatch(/^[a-zA-Z]+$/);
               }
             });
+            describe('alternativeModels', () => {
+              it('should only include entries defined in models json', () => {
+                for (const field in jsonData.fields) {
+                  if (Object.prototype.hasOwnProperty.call(jsonData.fields, field)) {
+                    const fieldSpec = jsonData.fields[field];
```

Comment:
alternatively:
```suggestion
                    const fieldSpec = jsonData.fields[field];
                    if (fieldSpec) {
                      continue;
                    }
```

Reply:
Thanks for the suggestion. The `hasOwnProperty` version is used many, many times in the file so I'll stick to that style for now.